visualization-server/rockcraft.yaml

```diff
@@ -1,41 +1,59 @@
-# Dockerfile: https://github.com/kubeflow/pipelines/blob/2.0.5/backend/Dockerfile.visualization
+# Based on: https://github.com/kubeflow/pipelines/blob/2.2.0/backend/Dockerfile.visualization
 name: visualization-server
-base: ubuntu@20.04
-version: '2.0.5'
 summary: ml-pipeline/visualization-server
 description: |
     ml-pipeline/visualization-server
     https://github.com/kubeflow/pipelines/tree/master/backend
-license: GPL-3.0
+version: "2.2.0"
+license: Apache-2.0
+base: ubuntu@22.04
 platforms:
     amd64:
 run-user: _daemon_
 services:
   vis-server:
     override: replace
-    command: python3.8 /server.py
+    summary: "visualization server service"
     startup: enabled
+    command: "python3 /server.py"
 
 package-repositories:
   - type: apt
     ppa: deadsnakes/ppa
     priority: always
 
 parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+      dpkg-query --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
+      > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
   python:
     plugin: python
     source: https://github.com/kubeflow/pipelines.git
     source-subdir: backend/src/apiserver/visualization
-    source-tag: 2.0.5
+    source-tag: 2.2.0
     stage-packages:
       # sync this python version to that in the tensorflow 
       # base image of Dockerfile.visualization.  Also keep this
       # in sync with PARTS_PYTHON_INTERPRETER below
     - python3.8-venv
+    - python3.8-dev
```

Comment:
Please correct me if I am wrong, but it looks like the python version might be 3.11, I checked that by going [here](https://github.com/kubeflow/pipelines/blob/2.2.0/backend/Dockerfile.visualization#L21) then checking that image's ptyhon [here](https://github.com/tensorflow/build/blob/master/tensorflow_runtime_dockerfiles/cpu.Dockerfile#L27).

Reply:
thats for the master .... the way how I tested is by checking upstream docker image for visualisation server 

```
❯ docker run --entrypoint /bin/bash gcr.io/ml-pipeline/visualization-server:2.2.0 -c "python --version"
Python 3.8.10
```